src/main/charts/jira/templates/statefulset.yaml

```diff
@@ -73,6 +73,8 @@ spec:
                 configMapKeyRef:
                   key: reserved_code_cache
                   name: {{ include "jira.fullname" . }}-jvm-config
+            - name: ATL_TOMCAT_CONTEXTPATH
```

Comment:
Should we fix Confluence chart too?

Reply:
I have created https://github.com/atlassian-labs/data-center-helm-charts/pull/90 to provide support for Confluence. The Bitbucket is still missing it as it provides access via Spring boot properties but I believe the `healthinessProbe` will still need to be changed.